4d0ff75b922447649057c237c0bd1545-epic.json

```diff
@@ -0,0 +1,7 @@
+{
+    "title": "The Elder Scrolls Online",
+    "notes": {
+        "dotnet48": "Install NET Framework 4.8"
```

Comment:
can you share what's the error that's solved with this? it's implicit that `dotnet48` installs .NET framework 4.8, but this notes should be something explaining what is the issue this solves so in the future we can check if this is still needed

something like `Fixes crash during launch` for example or `Fixes error window when clicking play in the third party launcher` or whatever the error is

Reply:
Game installer not even launch without this.